Fix InsecureSkipVerify in Dynatrace Plugin

thschue opened the pull request:
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

sjwang90 commented:
Issue https://github.com/influxdata/telegraf/issues/8014

ssoroka commented:
Rather than just removing the option, wouldn't it be better to add support for this and client certificates?

thschue commented:
> Rather than just removing the option, wouldn't it be better to add support for this and client certificates?

I think we implemented this by using the telegraf tls library (tls import and tls.ClientConfig in the struct, used in lines 253:264). This was also the cause for this Bug, as we switched to this without removing the "old" field.
